### PR TITLE
Add emojis

### PR DESCRIPTION
**Pull request by Naveen3Singh**
Added emojis as per [this feature request](https://github.com/SimpleMobileTools/Simple-Keyboard/issues/27)

### Notes:
- The view logic is the same as for the clipboard page.
- All emojis are shown together without any categorization.
- Emoji variations (skin tone, hair color, etc.) are ignored for now.

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/keyboard/media/emoji/Emoji.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.simplemobiletools.keyboard.media.emoji
+
+
+data class Emoji(val value: String, val name: String, val keywords: List<String>) {
+    override fun toString(): String {
+        return "Emoji { value=$value, name=$name, keywords=$keywords }"
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/keyboard/media/emoji/Emoji.kt:6]: I dont think we need this as it is a data class

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -31,8 +31,11 @@
     <string name="show_popup">إظهار نافذة منبثقة عند الضغط على المفاتيح</string>
     <string name="vibrate_on_keypress">يهتز عند ضغط الزر</string>
     <string name="keyboard_language">لغة لوحة المفاتيح</string>
+    <!-- Emojis -->
+    <string name="emojis">Emojis</string>
+    <string name="abc">ABC</string>
```
Comment (tibbi) [inline app/src/main/res/values-ar/strings.xml:36]: move this string to donottranslate

Comment (Naveen3Singh) [reply app/src/main/res/values-ar/strings.xml:36]: both or the "ABC" one?

Comment (tibbi) [reply app/src/main/res/values-ar/strings.xml:36]: just the ABC one

File: app/src/main/kotlin/com/simplemobiletools/keyboard/media/emoji/Emoji.kt (hunk 1 above)
Comment (Naveen3Singh) [reply app/src/main/kotlin/com/simplemobiletools/keyboard/media/emoji/Emoji.kt]: check pls

**Comment by tibbi** [comment]
clicking on the Emoji icon at the main keyboard vibrates twice instead of one

**Comment by tibbi** [comment]
and Clear at emojis doesnt vibrate at all

**Comment by tibbi** [comment]
emoji view should use the same Clear button as the one at casual keyboard

**Pull request merged**